index.html

```diff
@@ -7,227 +7,242 @@
-            <li>Bootstrap</li>
-            <li>Ruby on rails</li>
-            <li>Rspec</li>
-            <li>CopyBara</li>
-            <li>Selinium</li>
-          </ul>
-        </li>
-
-        <li>
-          <div class="lang-wrapper framework">
-            <img src="images/Rectangle 57.svg"class="rectangle57">
-            <h3 class="fhead">Skills</h3>
-          </div>
-            <ul class="language-list framework">
-                <li>Codekit</li> 
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-            </ul>
-        </li>
-  </ul>
-    </section>
-    <!-- contact me part-->
-    <section class="contact">
-      <div class="contact-us">
-        <p class="contactus-description">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-        <form action="https://formspree.io/f/meqdkpbp" method="post">
-          <ul class="contact-form">
-            <li><input type="text" name="name" placeholder="Full Name" maxlength="30" required></li>
-            <li><input type="email" name="email" placeholder="Email"  required></li>
-            <li><textarea type="text" name="message" id="" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea></li>
-            <li><button type="submit" class="contact-button" class="form-submit">Get in Touch</button></li>    
-         </ul>
-        </form>
-
+    <li>
+      <div class="lang-wrapper">
+        <img src="images/Rectangle 55.svg">
+        <h3 class="language">Language</h3>
       </div>
-      
-    </section>
-    <footer>
-        <ul class="footer-icon">
-          <li><a href="#"><img src="images/f-github.png" alt=""></a></li>
-          <li><a href="#"><img src="images/Group.png" alt=""></a></li>
-          <li><a href="#"><img src="images/f-what.png" alt=""></a></li>
-          <li><a href="#"><img src="images/f-twitter.png" alt=""></a></li>
-          <li><a href="#"><img src="images/f-M.png" alt=""></a></li>
-        </ul>
-        <hr>
-      </footer>
-  </div> 
+      <ul class="language-list">
+        <li>Javascript</li> 
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+      </ul>
+    </li>
+    <li>
+      <div class="lang-wrapper framework">
+        <img src="images/Rectangle 56.svg">
+        <h3 class="fhead">Frameworks</h3>
+      </div>
+      <ul class="language-list">
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>Rspec</li>
+        <li>CopyBara</li>
+        <li>Selinium</li>
+      </ul>
+    </li>
+    <li>
+      <div class="lang-wrapper framework">
+        <img src="images/Rectangle 57.svg">
+        <h3 class="fhead">Skills</h3>
+      </div>
+      <ul class="language-list framework">
+        <li>Codekit</li> 
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </li>
```

Comment:
please try to fix the font-style and size for your language section, your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200122509-65315b87-d6a0-45ff-be92-7315f9fec842.png)
figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/200122539-760199fa-ce52-48d4-9733-f2b7415e160b.png)